keyboards/nack/config.h

```diff
@@ -0,0 +1,59 @@
+/*
+Copyright 2020 pastapojken <pastapojken@gmail.com>
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/ 
+
+#pragma once
+#include "config_common.h"
+
+#define VENDOR_ID 0xFEED
```

Comment:
If you plan on adding VIA support in the future, I'd recommend finding an assigning a unique (in the QMK repo) VID now.

This tool can help your: https://yanfali.github.io/qmk_usb_usage/

Reply:
I was not aware of VIA, thank you for putting it on my radar.